main.js

```diff
@@ -5152,9 +5152,21 @@ function addProfile(profession, profile, base){
                 limit: 0
             };
             _vendorItems[_vendorItems.length] = {
-                pattern: /_Green_T[1-5]_Unid$/,
+                pattern: /_Green_T[1-7]_Unid$/,
                 limit: 0
             }; // Unidentified Green Gear
+            _vendorItems[_vendorItems.length] = {
+                pattern: /^Object_Trinket_/,
+                limit: 0
+            }; 
+             _vendorItems[_vendorItems.length] = {
+                pattern: /^Object_Skill_/,
+                limit: 0
+            };
+             _vendorItems[_vendorItems.length] = {
+                pattern: /^Magic Focus$/,

```

Comment:
The green gear, trinket and magic focus$ came from this thead:

https://greasyfork.org/en/forum/discussion/3861/x

but honestly I haven't seen a Magic Focus$ item.   I have used these additions to my script for about 2 months now with out issue and believe they are of good use.